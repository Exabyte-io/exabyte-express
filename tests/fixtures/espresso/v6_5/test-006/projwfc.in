version https://git-lfs.github.com/spec/v1
oid sha256:6d1664ae93e3ae0191d4ba06e74121fbaab432c32d343b4ef663e5c3d87ce859
size 158
