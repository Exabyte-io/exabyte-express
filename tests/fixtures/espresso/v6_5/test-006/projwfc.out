version https://git-lfs.github.com/spec/v1
oid sha256:ac8cf260cd52c3ea4b111f671d2346ca748c42214e1e5c085f9a0986234dad10
size 9873
